tests/test_klipthrupt.py

```diff
@@ -1104,191 +1099,6 @@ def test_psfsub_withklipandctmeas_rdi():
     assert pl_counts == pytest.approx(recovered_pl_counts_ktcorrected,rel = 0.05) 
 
 
-def test_psfsub_withKTandCTandCrop_adi():
```

Comment:
would it still be useful to check that KLIP throughput and core throughput can be run as part of the PSF subtraction process? If we need to crop down the data to make it faster to run, we can run the crop step function before running do_psf_subtraction.